packages/nimble-components/src/banner/template.ts

```diff
@@ -4,65 +4,70 @@ import { buttonTag } from '../button';
 import { iconExclamationMarkTag } from '../icons/exclamation-mark';
 import { iconInfoTag } from '../icons/info';
 import { iconTriangleFilledTag } from '../icons/triangle-filled';
-import { iconXmarkTag } from '../icons/xmark';
+import { iconTimesTag } from '../icons/times';
```

Comment:
I still hate that we're using a math-named icon rather than the more generic "x mark" for a close button. I kind of want to alias it or duplicate it with the name "dismiss" or something like that.

Reply:
I don't disagree that "times" isn't the most intuitive name for an icon used to close things. Our docs already list "close" as one of its metaphors, and we don't have any conventions around exporting the same icon with different names. So, I'm not sure there is much actionable to do with this. I'll leave the conversation open in case any other reviewers have thoughts on this.

Reply:
I agree it's poorly named, since names should describe the shape of the icon rather than their metaphors (unless there's only one relevant metaphor, which isn't the case here). I think the fix would be to rename it and provide good aliases rather than create a copy. But I'm not sure the best way to distinguish the two x icons. Maybe something like:
- `xmark` (status, fail) (unchanged from today)
- `xmark-2`or `xmark-thin` (times, close)

If we want to stage the deprecation we could temporarily duplicate it with a new name, but I'd want us to clean up the old copy once we've migrated important clients away from it. But it'd still be a breaking change to remove it.

I don't think any of that should be in this PR. If either of you feel strongly enough, let's file an issue and settle on names when we triage it.

Reply:
Filed #2139